aidial_assistant/json_stream/json_string.py

```diff
@@ -93,7 +91,3 @@ async def escape(stream: CharacterStream) -> str:
             # Ignore when model cannot escape text properly
```

Comment:
Why ignore? The reference to the "model" here look strange. The json parser shouldn't know anything about the producer of json. Maybe configure it via flag "relaxed_char_escaping"?